### ISSUE TITLE
Incorrect readings with firmware 2.4.2lt Beta and RAWv2 format

### ISSUE THREAD

**Issue by okko** [akx/ruuvidriver#4]
RuuviTag firmware 2.4.2lt Beta and RAWv2 shows incorrect readings:

```
{
"beefbeefbeef": {
  "dataFormat": 3,
  "rssi": -81,
  "humidity": 5.5,
  "temperature": -122.83,
  "pressure": 78104,
  "accelerationX": -13056,
  "accelerationY": -22527,
  "accelerationZ": -29444,
  "battery": 25768,
  "ts": 1568485996177
},
"vegevegevege": {
  "dataFormat": 3,
  "rssi": -85,
  "humidity": 5.5,
  "temperature": 50.85,
  "pressure": 60441,
  "accelerationX": 7935,
  "accelerationY": -1024,
  "accelerationZ": 15363,
  "battery": 59409,
  "ts": 1568485988683
}
}
```

The beefbeefbeef humidity should be about 53%, temp 15 degrees and pressure 1014 hPa, and vegevegevege humidity 54%, temp 14 degrees and pressure 1014 hPa.

**Comment by okko** [comment]
The dataFormat is 5 but gets parsed as 3.
https://github.com/ruuvi/ruuvi-sensor-protocols#data-format-5-protocol-specification-rawv2

**Comment by akx** [comment]
Closed via #5 and #6.
